experimental/packages/opentelemetry-exporter-metrics-otlp-http/test/node/CollectorMetricExporter.test.ts

```diff
@@ -190,6 +204,69 @@ describe('OTLPMetricExporter - node with json over http', () => {
       envSource.OTEL_EXPORTER_OTLP_METRICS_HEADERS = '';
       envSource.OTEL_EXPORTER_OTLP_HEADERS = '';
     });
+    it('should use delta temporality defined via env', () => {
+      {
+        envSource.OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE = 'delta';
```

Comment:
Might be better (simpler) to test with data tables:

```js
const deltaEnvs = ['delta', 'DELTA', ...];
for (const env of deltaEnvs) {
  envSource.OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE = env;
  const exporter = new OTLPMetricExporter();
  assert.strictEqual(exporter['_aggregationTemporalitySelector'], DeltaTemporalitySelector);
}
```

In this way, we don't have to repeat identical steps for each input every time.

Reply:
Done in [0dca329](https://github.com/open-telemetry/opentelemetry-js/pull/3305/commits/0dca3293a892f1ae66b18fcb01ff265e4d4e359b) :slightly_smiling_face: